cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/compute/GcpInstanceResourceBuilder.java

```diff
@@ -213,10 +217,15 @@ String getPublicKey(String groupPublicKey, String groupLoginUserName) {
         return publicKey;
     }
 
-    private String getHostname(CloudStack cloudStack, List<CloudResource> buildableResource) {
+    private String getHostname(CloudStack cloudStack, int privateId) {
```

Comment:
getHostName(Group group, int privateId) ?

Reply:
Nice suggestion! It simplified the code a lot. Thank you!